platforms/r3-corda/configuration/molecule/default/verify.yml

```diff
@@ -0,0 +1,157 @@
+---
+# This is an example playbook to execute Ansible tests.
+
+- name: Verify
+  hosts: localhost
+  tasks:
+
+  - name: Check if the storage class value file is created
+    stat:
+      path: "{{ playbook_dir }}/../../../releases/dev/aws-storageclass/aws-storageclass.yaml"
+    register: sc_check
+
+  - name: Assert if the value file exists or not
+    assert:
+      that: sc_check.stat.exists
+
+  - name: Deploy the resource on cluster
+    k8s:
+      state: present
+      src: "{{ playbook_dir }}/../../../releases/dev/aws-storageclass/aws-storageclass.yaml"
+  
+  - name: Wait for the storage class to come up
+    k8s_info:
+      kind: "StorageClass"
+      name: "aws-storageclass"
+      kubeconfig: "/tmp/molecule/kind-default/kubeconfig"
+      context: "kind"
+    register: sc_status
+    retries: 30
+    delay: 30
+
+  - name: Assert for storageclass creation
+    assert:
+      that: sc_status.resources|length > 0
+ 
+#######################################################################################
+
+  - name: Check if the namespace value file is created
+    stat:
+      path: "{{ playbook_dir }}/../../../releases/dev/test-ns/namespace.yaml"
+    register: ns_check
+
+  - name: Assert if the value file exists or not
+    assert:
+      that: ns_check.stat.exists
+
+  - name: Deploy the resource on cluster
+    k8s:
+      state: present
+      src: "{{ playbook_dir }}/../../../releases/dev/test-ns/namespace.yaml"
+  
+  - name: Wait for the namespace to come up
+    k8s_info:
+      kind: "Namespace"
+      name: "test-ns"
+      kubeconfig: "/tmp/molecule/kind-default/kubeconfig"
+      context: "kind"
+    register: ns_status
+    retries: 30
+    delay: 30
+
+  - name: Assert for namespace creation
+    assert:
+      that: ns_status.resources|length > 0
+
+####################################################################################
+
+  - name: Check if the vault-auth serviceaccount value file is created
+    stat:
+      path: "{{ playbook_dir }}/../../../releases/dev/test-ns/vaultAuth.yaml"
+    register: va_sa_check
+
+  - name: Assert if the value file exists or not
+    assert:
+      that: va_sa_check.stat.exists
+
+  - name: Deploy the resource on cluster
```

Comment:
same comment on all. Converge is when you run the test case (here trying to create the resources with the files generated), and verify is when you just check if the resources were created.